s4_reference_design.toml

```diff
@@ -20,39 +20,39 @@ CMB_tensor_to_scalar_ratio = 0.005
 
 total_experiment_length_years = 7
 
-# between 0 and 1, typically 0.2
-observing_efficiency.default = 0.2
-
-# we can override the observing efficiency for specific sites/bands
-# always specify first site, then telescope, then band
-observing_efficiency.Pole.SAT.HFS = 0.2
-observing_efficiency.Pole.LAT.HFL = 0.2
-
-# we can also specify defaults for sites (notice capitalization) or telescopes
-observing_efficiency.Pole.SAT.default = 0.2
-observing_efficiency.Chile.default = 0.2
 
 # Number of splits, 1 generates only full mission
 # 2-7 generates the full mission map and then the requested number
 # of splits scaled accordingly. E.g. 7 generates the full mission
 # map and 7 equal (yearly) maps
 number_of_splits = 1
 
-include_noise = false
 include_atmosphere = false
 
 # atmosphere maps scaling factors (multipled to input maps)
 atmosphere_scaling_T = 1.0
 atmosphere_scaling_P = 1.0
 
```

Comment:
@keskitalo @cbischoff do we want to scale the atmosphere?

Reply:
By number of days and observing efficiency yes. No scaling by detector count or number of optics tubes.

Reply:
I implemented this additional factor because you wanted to tweak those differently for T and P. Do we need it?

Reply:
We can start with 1.0 for both but I anticipate that we may want to adjust them later.